docs/configuration/caching.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Configure Codacy Self-hosted to use an external NFS server to improve the performance of the cloned repository cache.
+---
+
+# Caching
+
+Codacy Self-hosted includes a built-in NFS server provisioner that deploys a shared volume to cache the cloned repository files while they're being analyzed by each tool. However, if you're dealing with big repositories or a high volume of analysis, using an NFS server external to the cluster will improve the performance of the cache.
```

Comment:
Please check/validate the rationale for setting up an external NFS server instead of using the built-in one.